jstests/libs/backup_utils.js

```diff
@@ -16,6 +16,10 @@ function openBackupCursor(mongo) {
         try {
             return mongo.getDB("admin").aggregate([{$backupCursor: {}}]);
         } catch (exc) {
+            // Avoid infinite loop if Location31403
+            if ( exc.toString().includes("Location31403") ) {
```

Comment:
Or, as alternative - !exc.toString().includes("Location50915")

Reply:
> Or, as alternative - !exc.toString().includes("Location50915")
++ I like this approach better, since pbm currently can handle only Location50915 (overlap with checkpoint) error so anything else would be unexpected mongod behavior and basically we need to be aware of it.

Reply:
Ok, replace this to
```
!exc.toString().includes("Location50915")
```